_maps/holodeck/meeting.dmm

```diff
@@ -0,0 +1,456 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/obj/effect/turf_decal/siding/wood/corner{
+	dir = 1
+	},
+/turf/open/floor/carpet/royalblue,
+/area/template_noop)
+"b" = (
```

Comment:
why is this here? you need to add the template for this to show up in game if that was what you wanted